bin/opds_entry_coverage

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
See https://github.com/NYPL-Simplified/circulation-docker/pull/73 for this.

Reply:
Maybe you could make this a symlink to the script in core/bin and then you wouldn't have to duplicate it.

Reply:
I don't think a symlink would work because the import also needs to change.

Reply:
I'd have to test it but I think the core script would import relative to itself regardless of where you ran it from.

Reply:
I'll try it out.

Reply:
No, it doesn't work. :(

```
simplified/circulation$ bin/opds_entry_coverage
Traceback (most recent call last):
  File "bin/opds_entry_coverage", line 8, in <module>
    from coverage import OPDSEntryWorkCoverageProvider
ImportError: No module named coverage
```